scripts/dapp/browserify.js

```diff
@@ -95,16 +95,18 @@ gulp.task('browserify', async function(callback) {
           // underscore gets broken when we try to parse it
           /underscore/,
 
+
           // remove core-js and babel runtime,
           // https://github.com/babel/babel/issues/8731#issuecomment-426522500
-          /[\/\\]core-js/,
-          /@babel[\/\\]runtime/,
+         // /[\/\\]core-js/,
```

Comment:
better remove it completely or add some hint like "TODO: remove after tested" sow we know why it's still in the code.